Nishant/Day_12/postgres assignment/src/services/general.ts

```diff
@@ -0,0 +1,32 @@
+import {client as sqlclient} from '../database/db';
+
+export async function get_topper_by_classid_and_subjectid(c_id:number,s_id:number){
+    try{
+    await sqlclient.query("SET search_path TO College");
+          return (await sqlclient.query(`SELECT student_id,fname,S.marks FROM (SELECT * FROM result WHERE clas_id=${c_id} AND subjectid=${s_id} ORDER BY marks DESC) AS S,Student WHERE S.studentid=student_id LIMIT 1`)); 
+    }catch(e){
+        throw Error(e);
+    }
+}
+
+export async function get_top_students(c_id:number,num:number){
+    try{
+    await sqlclient.query("SET search_path TO College");
```

Comment:
we can use ORM's like https://joi.dev/api/?v=17.4.0 to do db operation. 
Try this out, no need to do this changes to this assignment